pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -33,11 +33,11 @@ const (
 	// clusterPlatformLabel is the label on a cluster deployment which indicates whether or not a cluster is on GCP platform
 	clusterPlatformLabel = "hive.openshift.io/cluster-platform"
 	clusterPlatformGCP   = "gcp"
-	orgParentFolderID    = "240634451310" // Service Delivery org subfolder
 
 	// secret information
 	gcpSecretName         = "gcp"
 	orgGcpSecretName      = "gcp-project-operator"
+	orgGcpConfigMap       = "gcp-project-operator"
```

Comment:
I think all the other variables are related to gcp secrets, and to avoid confusion they are prefixed with "org". This is not the case for the config map so maybe move it to the "Operator config" block above (line 27)